spec/rubocop/cop/style/method_name_spec.rb

```diff
@@ -49,6 +49,16 @@ def +(other)
       END
     end
 
+    it 'accepts unary operator definitions' do
+      expect_no_offenses(<<-END.strip_indent)
```

Comment:
I kind of like the convention to use `RUBY` instead o `END` for such heredocs. I think that in many regards the use of END everywhere is kind of an anti-patterns. People simply don't know they an use an more descriptive name. :-)